src/features/CourtCaseList/CourtCaseList.tsx

```diff
@@ -1,93 +1,38 @@
-import ColumnOrderIcons from "features/CourtCaseFilters/ColumnOrderIcons"
-import { Link, Paragraph, Table } from "govuk-react"
+import { Paragraph, Table } from "govuk-react"
 import { useRouter } from "next/router"
 import CourtCase from "services/entities/CourtCase"
 import User from "services/entities/User"
 import type { QueryOrder } from "types/CaseListQueryParams"
+import { useCustomStyles } from "../../../styles/customStyles"
 import CourtCaseListEntry from "./CourtCaseListEntry/CourtCaseListEntry"
+import { CourtCaseListTableHeader } from "./CourtCaseListTableHeader"
 
 interface Props {
   courtCases: CourtCase[]
   order?: QueryOrder
   currentUser: User
 }
 
-const CourtCaseList: React.FC<Props> = ({ courtCases, order = "asc", currentUser }: Props) => {
-  const { basePath, query } = useRouter()
+const CourtCaseList = ({ courtCases, order = "asc", currentUser }: Props) => {
```

Comment:
Do we want to make this function of type `React.FC<Props>` again? Doing so will tell us when we're passing something that doesn't match the expected argument types, and trying locally the compiler has no issue with the types

Reply:
I did some reading about it. I don't mind it either tbh. I think that TS still underlines wrong props in red if we leave it without `React.FC<Props>`.